### PR TITLE
[W5.7][F10-4] NEO KAI JUN

### PR DESCRIPTION
**Pull request by HyperionNKJ**
Added "size" command that allows user to check the number of persons in the address book.
Updated User guide as well.

### CODE REVIEW

**Comment by azhikai** [comment]
Hi Kai Jun,

Your commit titles shouldn't have any tenses (e.g. Updated, implemented). It should just be "Update size feature" and "Update user guide". Think of using the sentence "If this commit is merged,it will <your commit title>". So as an example, "If this commit is merged, it will implement size feature."

You may also wish to include some unit tests for your feature. 

Otherwise, all's good. 👍

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -156,6 +156,11 @@ Views all details of the 1st person in the results of the `find` command.
 Clears all entries from the address book. +
 Format: `clear`
 
+== Get size of address book : `size`
+
+Output the size of address book +
+Format: `size`
```
Comment (dalessr) [inline docs/UserGuide.adoc:162]: Keep it aligned with others (i.e. `Getting size of...`, `Outputs...`, full stop)

File: src/seedu/addressbook/commands/SizeCommand.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package seedu.addressbook.commands;
+/**
+ * Return the size of the address book
+ */
+public class SizeCommand extends Command {
+    public static final String COMMAND_WORD = "size";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Output the size of the address "
+            + "book "
```
Comment (dalessr) [inline src/seedu/addressbook/commands/SizeCommand.java:8]: Should add a `\n` behind?

File: src/seedu/addressbook/commands/SizeCommand.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+            + "Example: " + COMMAND_WORD;
+    public static final String MESSAGE_SUCCESS = "You have %d person(s) in your address book.";
+    @Override
+    public CommandResult execute() {
+        // Get the size of the addressbook
+        int size = addressBook.size();
+        // Return the command result
```
Comment (dalessr) [inline src/seedu/addressbook/commands/SizeCommand.java:15]: Are the comments really necessary here? The code can explain itself :)

File: src/seedu/addressbook/commands/SizeCommand.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        return new CommandResult(String.format(MESSAGE_SUCCESS, size));
+    }
+}
```
Comment (dalessr) [inline src/seedu/addressbook/commands/SizeCommand.java:18]: New line required at the end of each file

**Review by dalessr — APPROVED** [review]
@HyperionNKJ Well done for the `size` command 👍 Remember to add JUnit tests for the new feature accordingly.

Please close the PR after looking at the comments below.

File: src/seedu/addressbook/commands/SizeCommand.java (hunk 4 above)
Comment (HyperionNKJ) [reply src/seedu/addressbook/commands/SizeCommand.java:18]: Okay. Thanks for the review! :)

**Pull request closed without merging**